### PR TITLE
Additional style fixes for Utilization card

### PR DESCRIPTION
**Pull request by rawagner**
- make rows smaller
- fix padding
- fix card visual on smaller screens

before
ut_2_before
after
ut_2

before (no max values)
ut2_2_before

after
ut2_2

### CODE REVIEW

File: frontend/packages/console-shared/src/components/dashboard/utilization-card/utilization-card.scss [hunk 1]
```diff
@@ -36,14 +36,30 @@
...
 }
 
+.co-utilization-card__item-section {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.co-utilization-card__item-description {
```
Comment (rhamilto) [inline frontend/packages/console-shared/src/components/dashboard/utilization-card/utilization-card.scss]: Nit:  alphabetize rules by selector (this rule should come before the addition on line 50.

Comment (rawagner) [reply frontend/packages/console-shared/src/components/dashboard/utilization-card/utilization-card.scss]: fixed

**Pull request merged**